src/components/Movies/Movies.js

```diff
@@ -13,6 +13,7 @@ const Movies = ({ apiMovieData, handleMovieClick }) => {
   useEffect(() => {
     setMovies(apiMovieData);
     setNoMoviesFound(apiMovieData.length === 0);
+    document.body.style.overflow = 'visible'
```

Comment:
Interesting - didn't think we could use queries but if it makes it work that's great!